doc/manual/command-ref/nix-shell.xml

```diff
@@ -69,6 +69,11 @@ unpacked to a temporary location. The tarball must include a single
 top-level directory containing at least a file named
 <filename>default.nix</filename>.</para>
 
+<para>If <replaceable>path</replaceable> is of the form
+<literal>&lt;nixpkgs&gt;</literal>, it is replaced with the definition
+of <literal>nixpkgs</literal> in the environment variable
+<envar>NIX_PATH</envar>.</para>
```

Comment:
This is not a good description because it suggests that `<nixpkgs>` is special, while in fact it can be any path resolved via the Nix search path (e.g. `<foo/bar/xyzzy.nix>`). Also, the search path is not just determined by `NIX_PATH` but also by `-I` flags.